go.mod

```diff
@@ -20,6 +20,7 @@ replace (
 	github.com/cihub/seelog => github.com/cihub/seelog v0.0.0-20151216151435-d2c6e5aa9fbf // v2.6
 	github.com/coreos/go-systemd => github.com/coreos/go-systemd v0.0.0-20180202092358-40e2722dffea
 	github.com/docker/distribution => github.com/docker/distribution v2.7.1-0.20190104202606-0ac367fd6bee+incompatible
+	github.com/docker/docker => github.com/docker/docker v17.12.0-ce-rc1.0.20200916142827-bd33bbf0497b+incompatible
```

Comment:
We [should not introduce new `replace` directives on go.mod  if possible](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/gomodreplace.md). If we do, they need to be documented. Why do we need to add this one?

Reply:
There's a test that fails because the library introduced some breaking changes in the API.

I could try to fix that test, but I'm not sure if something else related with the docker integration could break. So I thought it was better to evaluate the docker client update separately in a different PR.

Reply:
I am okay with leaving it so long as there is a comment explaining why it is there. Note also that the `containers/image/v5` dependency seems to [expect the latest version](https://github.com/containers/image/blob/1895e312af410ccdee5efa44e5223ec93ae76001/go.mod#L13), so we should make sure that by `replac`ing docker by the older version we are not breaking `containers/images` in some suble way due to breaking changes.

Reply:
It turns out that we were only depending on `containers/image/v5` for a specific field that we don't need. So, I just deleted it. With that, we don't need to update docker :+1:

Reply:
Awesome!